curvefs/src/client/fuse_s3_client.cpp

```diff
@@ -118,15 +117,34 @@ void FuseS3Client::BackGroundFetch() {
         }
         {   // file need warmup
             std::list<fuse_ino_t> readAheadFiles;
-            readAheadFiles.swap(GetReadAheadFiles());
-            for (auto iter : readAheadFiles) {
-                VLOG(9) << "BackGroundFetch: " << iter;
-                fetchDataEnqueue(iter);
+            GetReadAheadFiles(&readAheadFiles);
+            if (readAheadFiles.empty()) {
+                VLOG(9) << "no readAheadFiles";
+            } else {
+                LOG(INFO) << "num of files is need loaded is: "
+                        << readAheadFiles.size();
+                for (auto iter : readAheadFiles) {
+                    VLOG(9) << "BackGroundFetch: " << iter;
+                    fetchDataEnqueue(iter);
+                }
+                LOG(INFO) << "num of files is loaded is: "
```

Comment:
What is the difference between this log and the above log

Reply:
deleted